doc/api/deprecations.md

```diff
@@ -634,6 +634,19 @@ Type: Runtime
 
 *Note*: change was made while `async_hooks` was an experimental API.
 
+<a id="DEP00XX"></a>
+### DEP00XX: crypto.createCipher()
+
+Type: End-of-Life
```

Comment:
This should not be automatically moved to `End-of-Life`. This needs to go through a proper deprecation cycle with either a Documentation-only or Runtime stage.